changelog/26663.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+ui: Fetch auth mount config values from `sys/internal/ui/mounts` endpoint for system default ttl value
```

Comment:
I might update this changelog to what it does for the user. eg. 
```suggestion
ui: Show computed values from `sys/internal/ui/mounts` endpoint for auth mount configuration view
```

Reply:
Great idea!